src/utils/DepositUtils.ts

```diff
@@ -122,6 +122,8 @@ export async function queryHistoricalDepositForFill(
     }
   }
 
+  deposit.messageHash ??= getMessageHash(deposit.message);
```

Comment:
I think you'll need something similar in `https://github.com/across-protocol/sdk/blob/f9a1e9c7753e83bdd666e89960daf76f6ff7cd67/src/utils/SpokeUtils.ts#L429` because it also makes a fresh event query.

Reply:
Good catch - https://github.com/across-protocol/sdk/pull/850/commits/7796919007fc7fe8237b7eef701de7d2adfa1e62